### DISCUSSION TITLE
How run it on windows?

### DISCUSSION THREAD

**Discussion by jonathanhecl** [PoloDB/PoloDB#142] [Q&A]
I don't understand, and isn't in the doc.

![image](https://github.com/PoloDB/PoloDB/assets/1691623/c07b6ec2-6f6b-4f0d-8e3f-9b2e7f4f22b8)

How create a localhost database? file and memory examples

thanks

**Comment by vincentdchan** [comment]
PoloDB is an embedded database. You should add it to your Rust program as a library and use through API. Thanks!
